app/src/main/res/layout/fragment_attach_photo.xml

```diff
@@ -11,9 +11,17 @@
         android:contentDescription="@string/quest_leave_new_note_photo"
         android:layout_centerVertical="true"/>
 
+    <TextView
+        android:id="@+id/photosAreUsefulExplanation"
+        android:layout_toEndOf="@id/takePhotoButton"
+        android:text="@string/quest_leave_new_note_photos_are_useful"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_centerVertical="true"/>
+
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/gridView"
-        android:layout_toEndOf="@id/takePhotoButton"
+        android:layout_toEndOf="@id/photosAreUsefulExplanation"
```

Comment:
It is problematic to place a view in relation to an element that is `View.GONE`. Not sure why it worked for you, usually is shouldn't. If the view are aligned next to each other, maybe consider a horizontal `LinearLayout` instead to solve this?

Reply:
That got carried over from the old version. What about both of views getting ` android:layout_toEndOf="@id/takePhotoButton"`? Always only one is displayed so I would expect it to work.